apm-agent-plugins/apm-mongodb/apm-mongodb4-plugin/src/main/resources/META-INF/services/org.stagemonitor.configuration.ConfigurationOptionProvider

```diff
@@ -0,0 +1 @@
+co.elastic.apm.agent.mongodb.MongoConfiguration
```

Comment:
[for reviewer] the reason this file is duplicated for 3.x and 4.x instrumentations and not created once in the common module is because those service definition files are used as heuristic to know which maven modules need to be added as dependencies in `apm-agent` (this is being tested by `PackagingTest.checkPluginDependencies`).